### PR TITLE
remove the symbolic link in the meson test

### PR DESCRIPTION
**Pull request by phlptp**
Remove the symbolic link for Issue #696

Add a symlink generation command in the meson CI test

### CODE REVIEW

**Comment by henryiii** [comment]
Calling it `.gitkeep` is somewhat standard. And you weren't able to mkdir it, I guess?

**Comment by phlptp** [comment]
trying the mkdir now

**Comment by phlptp** [comment]
looks like the mkdir from the command line works as well, so that is probably the best solution here

**Comment by jspelletier** [comment]
Note: the symlink should be deleted after the test or the problem with gitlab will persist if someone has a gitlab CI executing that test as a submodule. in my case since I am not executing the tests, I think that would work

**Comment by henryiii** [comment]
> gitlab CI executing that test as a submodule

Why would gitlab CI pick up a line from Azure pipelines?

File: azure-pipelines.yml [hunk 1]
```diff
@@ -82,15 +82,22 @@ jobs:
...
+      displayName: generate CLI11 symlink
     - script: meson build
       displayName: Run meson to generate build
       workingDirectory: tests/mesonTest
     - script: ninja -C tests/mesonTest/build
       displayName: Build with Ninja
     - script: ./tests/mesonTest/build/main --help
       displayName: Run help
+    - script: rm -rf tests/mesonTest/subprojects
+      displayName: remove symlinks
```
Comment (henryiii) [inline azure-pipelines.yml]: Cleaner without these lines, IMO.
```suggestion
```

**Review by henryiii — APPROVED** [review]

**Pull request merged**